Add negative test for PoW verification

alexfmpe opened the pull request.

alexfmpe committed:
Add negative test for PoW verification

alexfmpe committed:
Merge branch 'develop' into failing-pow

Ericson2314 commented on solana-bridges/solana-ethereum-client/src/tests.rs:
```diff
@@ -108,16 +108,22 @@ fn test_header_pow(header: &str) -> Result<(), TestError> {
     return Ok(());
 }
```
Delete the other `test_header_pow` which ends here?

Ericson2314 committed:
Fix some issues:

 - Can ignore a test rather than comment out #[test]

 - Remove duplicate function

 - Ignore unused only on test data, not all of tests
